<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>list</class>
 <widget class="QDialog" name="list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Sheets</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>421</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dlt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
